io.catenax.early_warning_notification/1.0.0/metadata.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
This file should only contain `{ "status" : "release"} `. please delete the content of this file.